### ISSUE TITLE
I can't open the onedrive link for DTU test set

### ISSUE THREAD

**Issue by yiping-tks** [Mercerai/PFGS#7]

**Comment by ShogoSato1996** [comment]
Same issue is happened.

**Comment by yiping-tks** [comment]
Copy that!

**Comment by ShogoSato1996** [comment]
Thank you for your reply!

Which link do you refer ?

**Comment by yiping-tks** [comment]
Oh, this is my github automatic response. For some reasons, I cant't open the onedrive link for the pre-trained weights and demo datasets.

> Copy that!

**Comment by ShogoSato1996** [comment]
Me too...
